src/UI/Seller/src/app/shared/components/user-group-assignments/user-group-assignments.component.ts

```diff
@@ -45,6 +46,7 @@ export class UserGroupAssignments implements OnChanges {
   @Output() hasAssignments = new EventEmitter<boolean>()
 
   userOrgID: string
+  buyerID = this.router.routerState.snapshot.url.split('/')[2]
```

Comment:
It seems like we would want this in an onInit/ somewhere in the flow so you can validate the url exists (and that the 3rd element exists)